### PR TITLE
remove an unnecessary gas param placeholder

### PR DESCRIPTION
**Pull request by zjma**
### Description

A dummy gas parameter was unnecessarily introduced by #7099 and is supposed to be deleted in #6550.
Here I'm fixing it in main anyway in case #6550 is not able to merge before the next branch cut.

### CODE REVIEW

File: aptos-move/framework/src/natives/cryptography/algebra/mod.rs [hunk 1]
```diff
@@ -289,7 +287,6 @@ fn deserialize_internal(
 
 macro_rules! ark_add_internal {
     ($gas_params:expr, $context:expr, $args:ident, $structure:expr, $ark_type:ty) => {{
-        $context.charge($gas_params.placeholder)?;
```
Comment (alinush) [inline aptos-move/framework/src/natives/cryptography/algebra/mod.rs:292]: Do you wanna add a todo comment here so we do not forget adding it back?

**Review by alinush — APPROVED** [review]
LGTM!

**Review by vgao1996 — APPROVED** [review]

File: aptos-move/framework/src/natives/cryptography/algebra/mod.rs (hunk 1 above)
Comment (zjma) [reply aptos-move/framework/src/natives/cryptography/algebra/mod.rs:292]: We don't have to. The full PR should fix everything.

Comment (alinush) [reply aptos-move/framework/src/natives/cryptography/algebra/mod.rs:292]: Can you add a note in the full PR?

**Pull request merged**